aws-encryption-sdk-cpp/tests/test_vectors/static_test_vectors.cpp

```diff
@@ -288,6 +289,7 @@ static aws_cryptosdk_keyring *keyring_for_master_key_spec(
             }
         }
     } else if (!cmp_jsonstr_with_cstr(key_type_obj, "aws-kms")) {
+        usleep(100);
```

Comment:
nit: let's pull this value out into a constant

Reply:
or better yet, since `keyring_for_master_key_spec` writes the key type into `test_type_idx`, we can move this delay logic into the caller in order to reduce duplication